cmd/scheduler.go

```diff
@@ -24,40 +21,16 @@ func addSchedulerCommand(a *app) *cobra.Command {
 			if cfg.Queue.Type != config.RedisQueueProvider {
 				log.WithError(err).Fatalf("Queue type error: Command is available for redis queue only.")
 			}
-			d, err := time.ParseDuration(timer)
-			if err != nil {
-				log.WithError(err).Fatalf("failed to parse time duration")
-			}
 
-			ticker := time.NewTicker(d)
-			ctx := context.Background()
+			s := worker.NewScheduler(&a.eventQueue)
 
-			for {
-				select {
-				case <-ticker.C:
-					go func() {
-						err := worker.RequeueEventDeliveries("Processing", timeInterval, a.eventDeliveryRepo, a.groupRepo, a.eventQueue)
-						if err != nil {
-							log.WithError(err).Errorf("Error requeuing status processing: %v", err)
-						}
-					}()
-					go func() {
-						err := worker.RequeueEventDeliveries("Scheduled", timeInterval, a.eventDeliveryRepo, a.groupRepo, a.eventQueue)
-						if err != nil {
-							log.WithError(err).Errorf("Error requeuing status Scheduled: %v", err)
-						}
-					}()
-					go func() {
-						err := worker.RequeueEventDeliveries("Retry", timeInterval, a.eventDeliveryRepo, a.groupRepo, a.eventQueue)
-						if err != nil {
-							log.WithError(err).Errorf("Error requeuing status Retry: %v", err)
-						}
-					}()
-				case <-ctx.Done():
-					ticker.Stop()
-					return
-				}
-			}
+			// Register tasks.
+			// s.AddTask("retry events", 30, func() {
+			// 	  task.RetryEventDeliveries(nil, "", a.eventDeliveryRepo, a.groupRepo, a.eventQueue)
+			// })
+
```

Comment:
Why are these commented out though?

Reply:
Because we don't need it again? I left it as a template for the next jobs we'll be adding.

Reply:
Oh ok